examples/channels-workers/channels_workers.bal

```diff
@@ -0,0 +1,34 @@
+import ballerina/io;
+//Defines a channel that accepts xml messages.
+channel<xml> xmlChn;
+
+public function main(string... args) {
+
+    worker w1 {
+        map key = {myKey:"abc123"};
+        io:println("w1 Waiting for a message...");
+        xml message;
+        //Waiting to receive a message with key from xmlChn channel.
+        message <- xmlChn, key;
+        io:println("w1 Received message: ", message);
+        io:println("w1 Waiting for a null key message...");
+        //Waiting to receive a message without key from xmlChn channel.
```

Comment:
Minor correction:
"//Wait to receive a message without a key from the xmlChn channel."